src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Generated/Models/EntityModel.cs

```diff
@@ -41,7 +41,7 @@ public EntityModel()
         /// matched entity within the utterance.</param>
         /// <param name="additionalProperties">Unmatched properties from the
         /// message are deserialized this collection</param>
-        public EntityModel(string entity, string type, double startIndex, double endIndex, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>))
+        public EntityModel(string entity, string type, int startIndex, int endIndex, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>))
```

Comment:
This is a breaking change. Please bump the major version number [here](https://github.com/Azure/azure-sdk-for-net/blob/2417f1146b412d8357555bc42378196b181cd712/src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.Runtime.csproj#L9) Also, update the `PackageReleaseNotes` accordingly

Reply:
Please update the `AssemblyInfo` [here](https://github.com/Azure/azure-sdk-for-net/blob/2417f1146b412d8357555bc42378196b181cd712/src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Properties/AssemblyInfo.cs) based on these version updates.

Reply:
@dsgouda where to update the PackageReleaseNotes?